index.ts

```diff
@@ -85,15 +88,21 @@ const touchInfo: {
 let touchSpeed = 0;
 let lastTouchAlpha = 0;
 
-function onTouchStart(e: TouchEvent) {
+function onTouchStart(e: TouchEvent | PointerEvent) {
+  let x = 'touches' in e ? (e as TouchEvent).touches[0].clientX : (e as PointerEvent).clientX;
+  let y = 'touches' in e ? (e as TouchEvent).touches[0].clientY : (e as PointerEvent).clientY;
+
   onTouchMove(e);
```

Comment:
Since we call this we could return x and y and reuse here. Then we don't need to duplicate these lines